spec/02-integration/09-hybrid_mode/09-forward-proxy_spec.lua

```diff
@@ -0,0 +1,176 @@
+local helpers = require "spec.helpers"
+local pl_path      = require "pl.path"
+local pl_file      = require "pl.file"
+
+
+local fixtures = {
+  stream_mock = {
+    forward_proxy = [[
+    server {
+      listen 16797;
+      error_log logs/proxy.log debug;
+
+      content_by_lua_block {
+        require("spec.fixtures.forward-proxy-server").connect()
+      }
+    }
+
+    server {
+      listen 16796;
+      error_log logs/proxy_auth.log debug;
+
+      content_by_lua_block {
+        require("spec.fixtures.forward-proxy-server").connect({
+          basic_auth = ngx.encode_base64("test:konghq"),
+        })
+      }
+    }
+
+    ]],
+  },
+}
+
+
+local confs = helpers.get_clustering_protocols()
```

Comment:
We are removing the legacy hybrid protocol in 3.1 (#9740), so thankfully this extra parameterization can be removed from your integration tests.